spec/zoneddatetime.html

```diff
@@ -0,0 +1,1045 @@
+    <ul>
+      <li>is the intrinsic object <dfn>%Temporal.ZonedDateTime.prototype%</dfn>.
+      <li>is itself an ordinary object.
+      <li>is not a Temporal.ZonedDateTime instance and does not have a [[InitializedTemporalZonedDateTime]] internal slot.
+      <li>has a [[Prototype]] internal slot whose value is %Object.prototype%.
+    </ul>
+
+    <emu-clause id="sec-temporal.zoneddatetime.prototype.constructor">
+      <h1>Temporal.ZonedDateTime.prototype.constructor</h1>
+      <p>The initial value of `Temporal.ZonedDateTime.prototype.constructor` is %Temporal.ZonedDateTime%.</p>
+    </emu-clause>
+
+    <emu-clause id="sec-temporal.zoneddatetime.prototype-@@tostringtag">
+      <h1>Temporal.ZonedDateTime.prototype[ @@toStringTag ]</h1>
+      <p>
+        The initial value of the @@toStringTag property is the string value `"Temporal.ZonedDateTime"`.
+      </p>
+      <p>
+        This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *true* }.
+      </p>
+    </emu-clause>
+
+    <emu-clause id="sec-get-temporal.zoneddatetime.prototype.calendar">
+      <h1>get Temporal.ZonedDateTime.prototype.calendar</h1>
+      <p>
+        `Temporal.ZonedDateTime.prototype.calendar` is an accessor property whose set accessor function is *undefined*.
+        Its get accessor function performs the following steps:
+      </p>
+      <emu-alg>
+        1. Let _zonedDateTime_ be the *this* value.
+        1. Perform ? RequireInternalSlot(_zonedDateTime_, [[InitializedTemporalZonedDateTime]]).
+        1. Return _zonedDateTime_.[[Calendar]].
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-get-temporal.zoneddatetime.prototype.timezone">
+      <h1>get Temporal.ZonedDateTime.prototype.timeZone</h1>
+      <p>
+        `Temporal.ZonedDateTime.prototype.timeZone` is an accessor property whose set accessor function is *undefined*.
+        Its get accessor function performs the following steps:
+      </p>
+      <emu-alg>
+        1. Let _zonedDateTime_ be the *this* value.
+        1. Perform ? RequireInternalSlot(_zonedDateTime_, [[InitializedTemporalZonedDateTime]]).
+        1. Return _zonedDateTime_.[[TimeZone]].
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-get-temporal.zoneddatetime.prototype.year">
+      <h1>get Temporal.ZonedDateTime.prototype.year</h1>
+      <p>
+        `Temporal.ZonedDateTime.prototype.year` is an accessor property whose set accessor function is *undefined*.
+        Its get accessor function performs the following steps:
+      </p>
+      <emu-alg>
+        1. Let _zonedDateTime_ be the *this* value.
+        1. Perform ? RequireInternalSlot(_zonedDateTime_, [[InitializedTemporalZonedDateTime]]).
+        1. Let _timeZone_ be _zonedDateTime_.[[TimeZone]].
+        1. Let _instant_ be ? CreateTemporalInstant(_zonedDateTime_.[[Nanoseconds]]).
+        1. Let _temporalDateTime_ be ? GetTemporalDateTimeFor(_timeZone_, _instant_).
```

Comment:
I think it'll be better to create the DateTime in the constructor and stick it in an internal slot. Followup fine.

Reply:
See explanation in https://github.com/tc39/proposal-temporal/pull/1044#discussion_r510968455 — the DateTime can't be cached unless the time zone is a built-in one and Temporal.TimeZone.prototype is %Temporal.TimeZone.prototype%.